test/functional/cmdLineTests/callsitedbgddrext/playlist.xml

```diff
@@ -30,7 +30,7 @@
 		</variations>
 		<command>$(MKTREE) $(REPORTDIR); \
 	cd $(REPORTDIR); \
-	perl $(JVM_TEST_ROOT)$(D)TestConfig$(D)scripts$(D)tools$(D)sysvcleanup.pl zos ; \
+	perl $(JVM_TEST_ROOT)$(D)TestConfig$(D)scripts$(D)tools$(D)sysvcleanup.pl all zos ; \
```

Comment:
Why are some tests changed `zos` -> `all zos`, while others are changed `zos` -> `all`?

Reply:
From https://github.com/eclipse/openj9/blob/master/test/TestConfig/scripts/tools/sysvcleanup.pl#62, first argument sets the `$cleanupuser` and second argument is used to set some variables specific to AIX and ZOS. Setting `$cleanupuser` to `all` ensures everything is deleted at https://github.com/eclipse/openj9/blob/master/test/TestConfig/scripts/tools/sysvcleanup.pl#L160, and there is no difference between setting the second argument to `sysvcleanup.pl` to zos or aix.

Therefore, `sysvcleanup` is called with first argument as `all` like in all other tests, and is called with second argument `zos` for the ZOS and AIX target.